guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -1,107 +1,101 @@
 ----
-# mkdir /mnt/_iso_
-# mount -o loop _rhel-binary-dvd.iso_ /mnt/_iso_
+# mkdir /mnt/iso
+# mount -o loop _rhel-binary-dvd.iso_ /mnt/iso
 ----
-
-ifeval::["{rhel-version}" == "8"]
-. Create directories for {RHEL} 8 AppStream and BaseOS Kickstart repositories:
 +
+Replace _rhel-binary-dvd.iso_ with the name of your ISO image.
+ifeval::["{el-major-version}" >= "8"]
+. Create directories for {RHEL} {el-major-version} AppStream and BaseOS Kickstart repositories:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart
-# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/baseos/kickstart
+# mkdir --parents /var/www/html/pub/{project-context}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/appstream/kickstart
+# mkdir --parents /var/www/html/pub/{project-context}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/baseos/kickstart
 ----
 endif::[]
 
-ifeval::["{rhel-version}" == "7"]
+ifeval::["{el-major-version}" == "7"]
 . Create Kickstart directories:
 +
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/
+# mkdir --parents /var/www/html/pub/{project-context}-import/content/dist/rhel/server/{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/kickstart/
 ----
 endif::[]
 
 . Copy the `kickstart` files from the ISO image:
 +
-ifeval::["{rhel-version}" == "8"]
-[options="nowrap" subs="+quotes"]
+ifeval::["{el-major-version}" >= "8"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# cp -a /mnt/_iso_/AppStream/* /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart
-
-# cp -a /mnt/_iso_/BaseOS/* /mnt/_iso_/images/ /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/baseos/kickstart
+# cp -a /mnt/iso/AppStream/* /var/www/html/pub/{project-context}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/appstream/kickstart
+# cp -a /mnt/iso/BaseOS/* /mnt/iso/images/ /var/www/html/pub/{project-context}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/baseos/kickstart
 ----
 +
-Note that for BaseOS, you must also copy the contents of the `/mnt/_iso_/images/` directory.
+Note that for BaseOS, you must also copy the contents of the `/mnt/iso/images/` directory.
 endif::[]
-ifeval::["{rhel-version}" == "7"]
-[options="nowrap" subs="+quotes"]
+ifeval::["{el-major-version}" == "7"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# cp -a /mnt/_iso_/* /var/www/html/pub/sat-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/
+# cp -a /mnt/iso/* /var/www/html/pub/{project-context}-import/content/dist/rhel/server/{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/kickstart/
```

Comment:
This needs to get resolved, as I believe this is an error made in the previous revision of this file. The step says to copy the contents of the `/mnt/iso/images` directory, but the actual command starts with `cp -a /mnt/iso/*` instead of `cp -a /mnt/iso/images/*`. Do you know someone who could confirm this?

Reply:
I don't know. Maybe @ekohl could kindly help?

Reply:
It was fairly easy to test and I tested with `rhel-server-7.9-x86_64-dvd.iso` and there the content of `images` looks a lot like what I'd expect in a kickstart tree:
```console
$ tree images
images
├── efiboot.img
├── pxeboot
│   ├── initrd.img
│   ├── TRANS.TBL
│   ├── upgrade.img
│   └── vmlinuz
└── TRANS.TBL

1 directory, 6 files
```

Reply:
so the cp command should be updated?

Reply:
Applied.